extra/resources/controld.in

```diff
@@ -64,20 +64,23 @@ In most cases, it should be run as an anonymous clone.
 
 <parameters>
 
-<parameter name="args" unique="1">
+<parameter name="args">
 <longdesc lang="en">
 Any additional options to start the dlm_controld service with
 </longdesc>
 <shortdesc lang="en">DLM Options</shortdesc>
 <content type="string" default="-s 0" />
 </parameter>
 
-<parameter name="daemon" unique="1">
+<parameter name="daemon" unique-group="daemon">
 <longdesc lang="en">
 The daemon to start - supports gfs_controld and dlm_controld
 </longdesc>
 <shortdesc lang="en">The daemon to start</shortdesc>
-<content type="string" default="dlm_controld" />
+<content type="select" default="dlm_controld">
+  <option value="gfs_controld" />
+  <option value="dlm_controld" />
+</content>
```

Comment:
Looking at this again, I don't think we restricted the daemon name before -- these were just the possible defaults

Theoretically someone could have a custom daemon under a different name

Reply:
Good catch